NateGoSearch/NateGoSearchQuery.php

```diff
@@ -234,10 +234,14 @@ public function query($keywords)
 		$misspellings = $this->getPopularReplacements($keywords, $misspellings);
 
 		$keywords = $this->normalizeKeywordsForSearching($keywords);
-		$keywords_hash = sha1($keywords);
+		$unique_hash = sha1($keywords.serialize($this->document_types));
```

Comment:
My bad, I read the code incorrectly. My only other question is should we make sure the shortnames are in order?